server/tests-py/test_scheduled_triggers.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+import pytest
+from datetime import datetime
+from datetime import timedelta
+from croniter import croniter
+from validate import validate_event_webhook,validate_event_headers
+from _queue import Empty
+import time
+
+def stringify_datetime(dt):
+    return dt.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+
+def get_events_of_scheduled_trigger(hge_ctx,trigger_name):
+    events_count_sql = '''
+    select count(*) from hdb_catalog.hdb_scheduled_events where name = '{}'
+    '''.format(trigger_name)
+    q = {
+        "type":"run_sql",
+        "args":{
+            "sql":events_count_sql
+        }
+    }
+    return hge_ctx.v1q(q)
+
+@pytest.mark.usefixtures("evts_webhook")
+class TestSubscriptionTrigger(object):
```

Comment:
why is it called `TestSubscriptionTrigger` ?

Reply:
Typo 😅